browser-test/src/support/applicant_questions.ts

```diff
@@ -430,6 +430,23 @@ export class ApplicantQuestions {
     return readFileSync(path, 'utf8')
   }
 
+  async downloadFileFromReviewPage(fileName: string) {
+    // Assert that we're on the review page.
+    expect(await this.page.innerText('h2')).toContain(
```

Comment:
**suggestion (non-blocking):** Playwright best practices suggest using the `page.getByRole()` locator because it's looking for it from a web first way that is the same as how accessibility tools look at it.  [getByRole](https://playwright.dev/docs/locators#locate-by-role).  In this case, I think you could do: `await expect(page.getByRole('heading', { name: 'Program application summary' })).toBeVisible();`